custom_components/solarman/inverter_definitions/deye_hybrid.yaml

```diff
@@ -280,6 +280,42 @@ parameters:
         offset: 1000
         registers: [0x00B6]
 
+      - name: "Grid Charge Ampere"
```

Comment:
1. Change Ampere to Current
2. This entity is controlling battery charging current from the grid? If so please name it in conformity with similar entity from for example SG04LP3 profile which is: "Battery Grid Charging Current"